packages/core/src/App/Containers/Modals/information-submitted-modal/information-submitted-modal.scss

```diff
@@ -0,0 +1,10 @@
+.information-submitted-modal {
+    @include mobile {
+        .dc-dialog {
+            &__content,
+            &__button {
+                font-size: 1.2rem;
```

Comment:
why do we need this?
Lets use Text component as pass this font size as prop

Reply:
Thanks for pointing this out. It's not needed. 👍